java/src/jmri/jmrit/display/layoutEditor/BlockValueFile.java

```diff
@@ -41,10 +44,13 @@ public BlockValueFile() {
     private Document doc = null;
     private Element root = null;
 
-    /*
+    /**
      *  Reads Block values from a file in the user's preferences directory
```

Comment:
Javadoc uses the first sentence as the brief description of a method, so you will want to add a period to the end of this sentence.

Reply:
Done.